encord/orm/label_row.py

```diff
@@ -307,4 +307,12 @@ def to_dict(self) -> dict:
         Returns:
             The dict equivalent of LabelRowMetadata.
         """
-        return asdict(self)
+
+        def transform(value: Any):
+            if isinstance(value, Enum):
+                return value.value
```

Comment:
life would be good if we could use pydantic here :)

Reply:
I honestly think we should add pydantic 😢